examples/README.md

```diff
@@ -41,4 +45,53 @@ Twitter provided a dataset for the [RecSys2020 challenge](http://www.recsyschall
 ### 5. [Applying the Techniques to other Tabular Problems with Rossmann](https://github.com/NVIDIA/NVTabular/tree/main/examples/tabular-data-rossmann) 
 
 Rossmann operates over 3,000 drug stores in 7 European countries. Historical sales data for 1,115 Rossmann stores are provided. The task is to forecast the "Sales" column for the test set. Kaggle hosted it as a [competition](https://www.kaggle.com/c/rossmann-store-sales/overview).
-- Using NVTabular for sales prediction
\ No newline at end of file
+- Using NVTabular for sales prediction
+
+## Start Examples
+
+You can run the examples by [installing NVTabular](https://github.com/NVIDIA/NVTabular#installation) and other required libraries. Alternativly, docker conatiners are available on http://ngc.nvidia.com/catalog/containers/ with pre-installed versions. Depending which example you want to run, you should select the docker container:
+- Merlin-Tensorflow-Training contains NVTabular with TensorFlow
+- Merlin-Pytorch-Training contains NVTabular with PyTorch
+- Merlin-Training contains NVTabular with HugeCTR
+- Merlin-Inference contains NVTabular with TensorFlow and Triton Inference support
+
+### Start Examples with Docker Container
+
+You can pull the container by running the following command:
+
+```
+docker run --runtime=nvidia --rm -it -p 8888:8888 -p 8797:8787 -p 8796:8786 --ipc=host --cap-add SYS_PTRACE <docker container> /bin/bash
```

Comment:
Maybe we should add a note about needing the nvidia docker runtime here? I believe @karlhigley ran into an issue following these because of this